packages/govern-core/test/govern.test.ts

```diff
@@ -39,6 +44,45 @@ describe('Govern', function () {
     ).to.be.revertedWith(ERRORS.AUTH)
   })
 
+  it('emits the ETHDeposited event as expected', async () => {
+    await expect(signers[0].sendTransaction(
+      {
+        value: 1000,
+        to: govern.address
+      }
+    ))
+      .to.emit(govern, EVENTS.ETH_DEPOSITED)
+      .withArgs(owner, 1000)
+  })
+
+  it('registers a standard and calls fallback as expected', async () => {
+    const magicNumber = '0x10000000'
+    const callbackSig = hexDataSlice(id('callbackFunc()'), 0, 4)
+
+    await expect(
+      govern.registerStandardAndCallback(
+        callbackSig,
+        callbackSig,
+        magicNumber
+      )
+    )
+      .to.emit(govern, EVENTS.REGISTERED_STANDARD)
+      .withArgs(callbackSig)
+      .to.emit(govern, EVENTS.REGISTERED_CALLBACK)
+      .withArgs(callbackSig, magicNumber)
+
+
+    await expect(signers[0].sendTransaction(
+      {
+        to: govern.address,
+        data: callbackSig
+      }
+      )
+    )
+    .to.emit(govern, EVENTS.RECEIVED_CALLBACK)
```

Comment:
Not sure if possible with a low level return, but maybe we can also assert the right `magicNumber` got returned.

Reply:
I think we can add this test case in the ``AdaptativeERC165`` test where it actually belongs to or we would duplicate the test cases too much. I was just not able to mock the extended contract.